2019-11-20 21:39:57,254 : kivy : Factory: 184 symbols loaded
2019-11-20 21:39:57,349 : kivy : Image: Providers: img_tex, img_dds, img_sdl2, img_pil, img_gif (img_ffpyplayer ignored)
2019-11-20 21:39:57,398 : kivy : VideoGstplayer: Using Gstreamer 1.16.0.0
2019-11-20 21:39:57,399 : kivy : Video: Provider: gstplayer
2019-11-20 21:39:57,464 : kivy : Loading MPF-MC controller
2019-11-20 21:39:57,464 : mpfmc : Mission Pinball Framework Media Controller v0.53.0-dev.15
2019-11-20 21:39:57,464 : mpfmc : Mission Pinball Framework Game Engine v0.53.0-dev.68
2019-11-20 21:39:57,464 : mpfmc : Machine path: D:\mpf\wows\wows
2019-11-20 21:39:57,465 : ConfigProcessor : Config file in cache changed: D:\mpf\wows\wows\config\config
2019-11-20 21:39:57,466 : ConfigProcessor : Loading config from file c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\mcconfig.yaml.
2019-11-20 21:39:57,511 : ConfigProcessor : Loading config: c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\mcconfig.yaml
2019-11-20 21:39:57,518 : ConfigProcessor : Loading config from file D:\mpf\wows\wows\config\config.
2019-11-20 21:39:57,601 : ConfigProcessor : Loading config: D:\mpf\wows\wows\config\config
2019-11-20 21:39:57,603 : ConfigProcessor : Config file cache created: C:\Users\schol\AppData\Local\Temp\31418c17c7947b0b26bf3ae2a4803514.mpf_cache
2019-11-20 21:39:57,603 : mpfmc : Starting clock at 60.0Hz
2019-11-20 21:39:57,616 : kivy : Text: Provider: sdl2
2019-11-20 21:39:57,654 : kivy : Window: Provider: sdl2
2019-11-20 21:39:58,021 : kivy : GL: Using the "OpenGL" graphics system
2019-11-20 21:39:58,024 : kivy : GL: GLEW initialization succeeded
2019-11-20 21:39:58,024 : kivy : GL: Backend used <glew>
2019-11-20 21:39:58,024 : kivy : GL: OpenGL version <b'4.6.0 NVIDIA 388.13'>
2019-11-20 21:39:58,024 : kivy : GL: OpenGL vendor <b'NVIDIA Corporation'>
2019-11-20 21:39:58,024 : kivy : GL: OpenGL renderer <b'GeForce GTX 1060 6GB/PCIe/SSE2'>
2019-11-20 21:39:58,024 : kivy : GL: OpenGL parsed version: 4, 6
2019-11-20 21:39:58,024 : kivy : GL: Shading version <b'4.60 NVIDIA'>
2019-11-20 21:39:58,024 : kivy : GL: Texture max size <32768>
2019-11-20 21:39:58,024 : kivy : GL: Texture max units <32>
2019-11-20 21:39:58,039 : kivy : Window: auto add sdl2 input provider
2019-11-20 21:39:58,040 : kivy : Window: virtual keyboard not allowed, single mode, not docked
2019-11-20 21:39:58,047 : SoundSystem : SoundSystem: Using default 'sound_system' settings
2019-11-20 21:39:58,226 : AudioInterface : Initialized
2019-11-20 21:39:58,227 : SoundSystem : No audio tracks are specified in your machine config file. a track named 'default' has been created.
2019-11-20 21:39:58,228 : kivy : Base: Start application main loop
2019-11-20 21:39:58,229 : EventManager : Event: ======'display_window_initialized'====== Args={}
2019-11-20 21:39:58,229 : EventManager : Event: ======'display_dmd_initialized'====== Args={}
2019-11-20 21:39:58,268 : mpfmc : Display: Setting 'dmd' as default display
2019-11-20 21:39:58,273 : EventManager : Event: ======'displays_initialized'====== Args={}
2019-11-20 21:39:58,274 : EventManager : Event: ======'slide_window_blank_created'====== Args={}
2019-11-20 21:39:58,274 : EventManager : Event: ======'display_window_ready'====== Args={}
2019-11-20 21:39:58,275 : EventManager : Event: ======'slide_dmd_blank_created'====== Args={}
2019-11-20 21:39:58,275 : EventManager : Event: ======'display_dmd_ready'====== Args={}
2019-11-20 21:39:58,284 : EventManager : Event: ======'init_phase_1'====== Args={}
2019-11-20 21:39:58,286 : EventManager : Event: ======'mc_ready'====== Args={}
2019-11-20 21:39:58,287 : EventManager : Event: ======'init_phase_2'====== Args={}
2019-11-20 21:39:58,287 : ConfigProcessor : Loading config from cache: C:\Users\schol\AppData\Local\Temp\d5fd805c5432c1441bf01a6268536a3d.mpf_cache
2019-11-20 21:39:58,288 : ConfigProcessor : Loading config from cache: C:\Users\schol\AppData\Local\Temp\5c4497fcd69d1d656c4b69a28e44c490.mpf_cache
2019-11-20 21:39:58,288 : ConfigProcessor : Loading config from cache: C:\Users\schol\AppData\Local\Temp\a327e3a062200f866ce963b9d13b0dd9.mpf_cache
2019-11-20 21:39:58,289 : ConfigProcessor : Loading config from cache: C:\Users\schol\AppData\Local\Temp\0aed3b95ee139e53e9a8d065ebb21707.mpf_cache
2019-11-20 21:39:58,289 : ConfigValidator : Your config contains a value for the setting "widgets:text:widgets:text:transition", but this is not a valid setting name.
2019-11-20 21:39:58,289 : kivy : Base: Leaving application in progress...
2019-11-20 21:39:58,289 : kivy : Exception while processing RegisteredHandler(callback=<bound method ModeController._load_modes of <mpfmc.core.mode_controller.ModeController object at 0x000001E8EE3FAE80>>, priority=1, kwargs={}, key=UUID('8806b364-82d6-40b9-b5d3-e48a14cac19f'), condition=None, blocking_facility=None) for event init_phase_2
Traceback (most recent call last):
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py", line 662, in _run_handlers
    result = handler.callback(**merged_kwargs)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\core\mode_controller.py", line 59, in _load_modes
    self.mc.modes[mode] = self._load_mode(mode)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\core\mode_controller.py", line 140, in _load_mode
    return Mode(self.mc, config, mode_string, mode_path)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\core\mode.py", line 46, in __init__
    **item.kwargs)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\core\config_collection.py", line 50, in create_entries
    self[name] = self.process_config(settings)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\config_collections\slide.py", line 33, in process_config
    config['widgets'][i] = self.mc.widgets.process_widget(widget)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\config_collections\widget.py", line 76, in process_widget
    config['type']).lower(), config, base_spec='widgets:common')
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_validator.py", line 251, in validate_config
    validation_failure_info)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_validator.py", line 418, in check_for_invalid_sections
    'setting name.', 2, self.log.name)
mpf.exceptions.config_file_error.ConfigFileError: Config File Error in ConfigValidator: Your config contains a value for the setting "widgets:text:widgets:text:transition", but this is not a valid setting name. Error Code: CFE-ConfigValidator-2 (https://docs.missionpinball.org/en/dev/logs/CFE-ConfigValidator-2.html)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\commands\mc.py", line 213, in __init__
    thread_stopper=thread_stopper).run()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\kivy\app.py", line 855, in run
    runTouchApp()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\kivy\base.py", line 504, in runTouchApp
    EventLoop.window.mainloop()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\kivy\core\window\window_sdl2.py", line 747, in mainloop
    self._mainloop()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\kivy\core\window\window_sdl2.py", line 479, in _mainloop
    EventLoop.idle()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\kivy\base.py", line 339, in idle
    Clock.tick()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\kivy\clock.py", line 591, in tick
    self._process_events()
  File "kivy\_clock.pyx", line 384, in kivy._clock.CyClockBase._process_events
  File "kivy\_clock.pyx", line 414, in kivy._clock.CyClockBase._process_events
  File "kivy\_clock.pyx", line 412, in kivy._clock.CyClockBase._process_events
  File "kivy\_clock.pyx", line 167, in kivy._clock.ClockEvent.tick
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\uix\display.py", line 205, in _displays_initialized
    self.mc.displays_initialized()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\core\mc.py", line 336, in displays_initialized
    self._init()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\core\mc.py", line 380, in _init
    self.events.process_event_queue()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py", line 743, in process_event_queue
    **event[3])
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py", line 708, in _process_event
    result = self._run_handlers(event, ev_type, kwargs)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py", line 664, in _run_handlers
    raise Exception("Exception while processing {} for event {}".format(handler, event)) from e
Exception: Exception while processing RegisteredHandler(callback=<bound method ModeController._load_modes of <mpfmc.core.mode_controller.ModeController object at 0x000001E8EE3FAE80>>, priority=1, kwargs={}, key=UUID('8806b364-82d6-40b9-b5d3-e48a14cac19f'), condition=None, blocking_facility=None) for event init_phase_2
2019-11-20 21:39:58,293 : kivy : Stopping child threads... (1 remaining)
2019-11-20 21:39:59,299 : kivy : All child threads stopped.
